src/core/control/Control.cpp

```diff
@@ -1147,6 +1154,18 @@ void Control::selectFillAlpha(bool pen) {
     }
 }
 
+void Control::selectPageMoveTo() {
+    MovePageDialog dlg(gladeSearchPath, getCurrentPageNo(), this->doc->getPageCount());
+
+    dlg.show(GTK_WINDOW(this->win->getWindow()));
+    auto oldPage = dlg.getSelectedPageFrom();
+    auto newPage = dlg.getSelectedPageTo();
+
+    if (oldPage > 0 && newPage > 0) {
+        this->movePage(size_t(oldPage - 1), size_t(newPage - 1));
+    }
```

Comment:
This should be asserted, so that a putative bug would not go unnoticed.
```suggestion
    assert(oldPage > 0 && newPage > 0);
    this->movePage(size_t(oldPage - 1), size_t(newPage - 1));
```